plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -837,23 +843,24 @@ void GSRendererHW::SwSpriteRender()
 				// Blending
 				ASSERT(m_context->ALPHA.A == 0);
 				ASSERT(alpha_b == 1 || alpha_b == 2);
-				ASSERT(m_context->ALPHA.C == 0);
 				ASSERT(m_context->ALPHA.D == 1);
-				ASSERT(m_context->ALPHA.FIX == 0);
 
-				GSVector4i sc_alpha_vec =
-					sc.yyww()   // 0x00AA00BB00AA00BB00aa00bb00aa00bb
+				// Flag C
+				GSVector4i sc_alpha_vec = alpha_c == 2 ?
+					GSVector4i(alpha_fix).xxxx().ps32() :
+					(alpha_c == 0 ? sc : dc0)
+					.yyww()     // 0x00AA00BB00AA00BB00aa00bb00aa00bb
```

Comment:
Ditto.

Reply:
I avoid the ternary operator for the ALPHA.C = 2 case, but maybe for 0 and 1 I will keep it